### PR TITLE
Add restarting

### PR DESCRIPTION
**Pull request by DannyLeWasTaken**
- Simply add /restart which will shutdown the bot, but thanks to the docker setup should in theory start back up.

### CODE REVIEW

File: src/commands/restart.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Basic ping bot
+
```
Comment (Nomalah) [inline src/commands/restart.ts:2]: Typo

**Review by Nomalah — CHANGES_REQUESTED** [review]
Put a short description of the command in the readme.
If you haven't already, it would be nice if you would install docker, run it locally using the command in the deploy job in the workflow, and then check if it works. Otherwise we'll just yolo it

**Comment by DannyLeWasTaken** [comment]
Tried to get docker up and running but it really doesn't like my PC.

**Review by Nomalah — APPROVED** [review]
Put a short description of the command in the readme.

**Pull request merged**